packages/nextra/src/server/loader.ts

```diff
@@ -131,12 +131,12 @@ ${themeConfigImport && '__nextra_internal__.themeConfig = __themeConfig'}`
   const locale =
     locales[0] === '' ? '' : mdxPath.replace(PAGES_DIR, '').split('/')[1]
 
-  const route =
+  const route = slash(
     '/' +
     path
       .relative(PAGES_DIR, mdxPath)
```

Comment:
```suggestion
    slash(path
      .relative(PAGES_DIR, mdxPath))
```
we don't need wrap `.replace(..)` with slash